test/e2e/cluster.sh

```diff
@@ -159,6 +172,9 @@ if [ "${UPGRADE_CLUSTER}" = "true" ] || [ "${SCALE_CLUSTER}" = "true" ] || [ -n
   fi
 
   if [ "${GET_CLUSTER_LOGS}" = "true" ]; then
+      if [ -z "${PRIVATE_SSH_KEY}" ]; then
```

Comment:
PRIVATE_SSH_KEY ="${PRIVATE_SSH_KEY:-_output/${RESOURCE_GROUP}-ssh}"

would this work? I would move close to the public key code